shell/ptty_windows.go

```diff
@@ -11,8 +12,31 @@ import (
 	"github.com/gruntwork-io/terragrunt/options"
 )
 
+// PrepareConsole enables support for escape sequences
+// https://stackoverflow.com/questions/56460651/golang-fmt-print-033c-and-fmt-print-x1bc-are-not-clearing-screenansi-es
+// https://github.com/containerd/console/blob/f652dc3/console_windows.go#L46
+func PrepareConsole(terragruntOptions *options.TerragruntOptions) {
+	enableVirtualTerminalProcessing(terragruntOptions, os.Stdin)
+	enableVirtualTerminalProcessing(terragruntOptions, os.Stderr)
+	enableVirtualTerminalProcessing(terragruntOptions, os.Stdout)
+}
+
+func enableVirtualTerminalProcessing(options *options.TerragruntOptions, file *os.File) {
+	var mode uint32
+	handle := windows.Handle(file.Fd())
+	if err := windows.GetConsoleMode(handle, &mode); err != nil {
+		options.Logger.Printf("failed to get console mode: %v\n", err)
+		return
+	}
+
+	if err := windows.SetConsoleMode(handle, mode|windows.ENABLE_VIRTUAL_TERMINAL_PROCESSING); err != nil {
+		options.Logger.Printf("failed to set console mode: %v\n", err)
+		_ = windows.SetConsoleMode(handle, mode)
```

Comment:
NIT: If we're going to ignore the error, you don't need the `_ = ` portion.

Reply:
Sorry about that - GoLand without it was warning me that error is ignored

Reply:
But you're still ignoring the error, so how is this better? Either don't use it at all or at least log it...